db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -669,6 +669,33 @@ private void getBuiltinSystemSchemas(){
 
     }
 
+    public List<SchemaDescriptor> getAllSchemas(TransactionController tc) throws StandardException{
+        List<SchemaDescriptor> lists =new ArrayList<>();
+        TabInfoImpl ti=coreInfo[SYSSCHEMAS_CORE_NUM];
+        SYSSCHEMASRowFactory rf=(SYSSCHEMASRowFactory)ti.getCatalogRowFactory();
+        ScanController sc=tc.openScan(
+                ti.getHeapConglomerate(),
+                false,   // don't hold open across commit
```

Comment:
Can you replace this and the following comments with their corresponding constants (or local values) and use them instead, e.g.

```
boolean dontHoldOpenAcrossCommit = false
```